app.py

```diff
@@ -55,6 +65,61 @@ def func():
 def get_games():
     return jsonify(games), 200
 
+@app.route('/api/tournaments', methods=['GET'])
+def get_tournaments():
+    return jsonify(tournaments), 200
+
+
+@app.route('/api/teams', methods=['GET'])
+def get_all_teams():
+    pub_teams = []
+    teams = Team.query.all()
+
+    for team in teams:
+        pub_teams.append(team.as_pub_dict())
+    return jsonify({'teams': pub_teams}), 200
+
+
+#@app.route('/api/teams/<team_name>')
+
+
+# TODO check if the game exists before creating a team
+@app.route('/api/teams/<game>/<name>', methods=['POST'])
```

Comment:
http://stackoverflow.com/questions/611906/http-post-with-url-query-parameters-good-idea-or-not

si ca peut vous faires changer d'avis